doc/articles/platform-specific-csharp.md

```diff
@@ -32,9 +32,9 @@ The structure of an Uno app created with the default [Visual Studio template](ht
  | UWP         | `NETFX_CORE`  |
  | Android     | `__ANDROID__` |
  | iOS         | `__IOS__`     |
- | WebAssembly | `__WASM__`    |
+ | WebAssembly | `HAS_UNO_WASM`|
```

Comment:
Huh! Why?

Reply:
Both are valid, but I'd rather to use a more uno-specific one (aligned with what the SKIA runtime is exposing).

Reply:
That one is new to me 👍